packages/web-components/src/components/table-of-contents/__stories__/README.stories.mdx

```diff
@@ -30,8 +30,15 @@ import '@carbon/ibmdotcom-web-components/es/components/table-of-contents/table-o
 ```scss
 @import '@carbon/ibmdotcom-styles/scss/globals/vars';
 @import '@carbon/ibmdotcom-styles/scss/components/tableofcontents/tableofcontents';
+
+body {
+  scroll-behaviour: smooth;
+  overflow-y: scroll;
```

Comment:
I'm wondering if this will mess with the masthead megamenu behavior. When the megamenu is open, we disable scroll in the body tag

Reply:
I guess it should be fine since we have the class to disable it